blast/process_genbank_db.py

```diff
@@ -63,9 +115,11 @@ def get_source_qualifiers(source_qual):
             else: 
                 quals.append(None)
         return quals
+
     #Get qualifiers from 'source' feature
     source_index = get_source_index(gb_entry.features)
     source_qualifiers = get_source_qualifiers(gb_entry.features[source_index].qualifiers)
+
     #Order: accession id, sequence length, description, organism, isolate, host, country
     metadata = (
```

Comment:
would a dictionary be better in this case?
line 158 iterates through it anyways

programmatically (and maybe for clarity), it may make more sense to use a for loop to go through and just say "for each characteristic in the given dictionary, append this to the metadata"

Reply:
I'll figure out why I was trolling here later